electrum/interface.py

```diff
@@ -1019,8 +1019,8 @@ async def get_balance_for_scripthash(self, sh: str) -> dict:
         # check response
         assert_dict_contains_field(res, field_name='confirmed')
         assert_dict_contains_field(res, field_name='unconfirmed')
-        assert_non_negative_integer(res['confirmed'])
-        assert_non_negative_integer(res['unconfirmed'])
+        assert_integer(res['confirmed'])
```

Comment:
Technically I believe one can assert that confirmed is non-negative, since it must be in "MoneyRange" which is all non-negative integers. Author's choice here.  Of course just asserting integer is not wrong either.. depends on how constraining one wants to be here.